code/modules/power/laser/PTL.dm

```diff
@@ -6,21 +6,14 @@
 	idle_power_usage = 1000
 	active_power_usage = 1000000
 	var/laser_beam_strength = 1000000
-
-
-
-
-
-
-
+	var/list/laser_tile_x_offset = list("north" = 0, "south" = 0, "east" = 2, "west" = -2)	//Depends on the sprite, right now it's goon's, so 3x3.
```

Comment:
```NORTH``` is ```#define NORTH 1```
```[]``` in a string is to embed
```""``` is a string

All three together turns NORTH->1->"1", this is MUCH better because you can then just do:
```laser_tile_x_offset["[someDir]"]``` without having to convert ```someDir```